src/EditorFeatures/CSharpTest/TypeInferrer/TypeInferrerTests.cs

```diff
@@ -22,8 +22,8 @@ protected override void TestWorker(Document document, TextSpan textSpan, string
             var typeInference = document.GetLanguageService<ITypeInferenceService>();
 
             var inferredType = useNodeStartPosition
-                ? typeInference.InferType(document.GetSemanticModelForSpanAsync(new TextSpan(node.SpanStart, 0), CancellationToken.None).Result, node.SpanStart, objectAsDefault: true, cancellationToken: CancellationToken.None)
-                : typeInference.InferType(document.GetSemanticModelForSpanAsync(node.Span, CancellationToken.None).Result, node, objectAsDefault: true, cancellationToken: CancellationToken.None);
+                ? typeInference.InferType(document.GetSemanticModelForSpanAsync(new TextSpan(node?.SpanStart ?? textSpan.Start, 0), CancellationToken.None).Result, node?.SpanStart ?? textSpan.Start, objectAsDefault: true, cancellationToken: CancellationToken.None)
+                : typeInference.InferType(document.GetSemanticModelForSpanAsync(node?.Span ?? textSpan, CancellationToken.None).Result, node, objectAsDefault: true, cancellationToken: CancellationToken.None);
```

Comment:
What are these changes about?  Also, would it maybe be clearer to just have:
```
textSpan = node?.Span ?? textSpan;
```

Before this, so that you can unconditionally use textSpan?